worklets/Overview.bs

```diff
@@ -262,49 +265,83 @@ is empty; it is populated when the user agent chooses to create its {{WorkletGlo
 A {{Worklet}} has a list of the <dfn>worklet's resolved module URLs</dfn>. Initially this list is
 empty; it is populated when module scripts resolved.
 
+A {{Worklet}} has a <dfn>fetched modules map</dfn>. This is a map of <a>fetch</a> requests to values
+that are a <a>fetch</a> responses. Access to this map should be thread-safe.
+
+The <a>fetched modules map</a> exists to ensure that {{WorkletGlobalScope}}s created at different
+times contain the same set of scripts and have the same behaviour. The creation of additional
```

Comment:
"same set of scripts" is a bit ambiguous, since at least in the spec sense, they have different scripts. It's more about using the same source text, and not doing double-fetches, I believe.